test/fixtures/recordings/Index-Tests_647266286/wrap-function-supports-request-options-check-with-functions_4061816471/recording.har

```diff
@@ -4,44 +4,62 @@
     "creator": {
       "comment": "persister:fs",
       "name": "Polly.JS",
-      "version": "3.0.0"
+      "version": "5.0.0"
     },
     "entries": [
       {
-        "_id": "f5898a5d0d94f177a8354428e4cbcd98",
+        "_id": "c9f4571142c99f547fd078ac983c98bb",
         "_order": 0,
         "cache": {},
         "request": {
           "bodySize": 0,
           "cookies": [],
           "headers": [
             {
-              "name": "referer"
+              "name": "connection",
+              "value": "keep-alive"
             },
             {
-              "name": "host",
-              "value": "www.example.com"
+              "name": "accept",
+              "value": "application/json,text/*;q=0.9,*/*;q=0.8"
+            },
+            {
+              "name": "user-agent",
+              "value": "helix-status/7.1.3 (@adobe/helix-status@7.1.3)"
+            },
+            {
+              "name": "accept-encoding",
+              "value": "br;q=1, gzip;q=0.8, deflate;q=0.5"
             },
             {
-              "name": "content-length",
-              "value": 0
+              "name": "referer",
+              "value": "/_status_check/healthcheck.json"
```

Comment:
This is strange.

Reply:
why? I changed how the router is mounted.